### PR TITLE
Fixing connection errors from edx forum to AWS managed Elasticsearch

### PR DESCRIPTION
**Pull request by blarghmatey**
- Added an open access policy to allow for anonymous communication from inside the VPC
- Added Forum configuration parameters to use the logic added in a commit to our fork
- Updated the Packer AMI to build with our fork of the forum code

### CODE REVIEW

**Comment by shaidar** [comment]
Is the only way to go about not signing every request and using a proxy is allowing anonymous comm from the VPC or is there some way to restrict it a tiny bit more, maybe by using some IP based restrictions?

File: src/bilder/images/edxapp/templates/edxapp/mitx-staging/forum.env [hunk 1]
```diff
@@ -12,6 +12,8 @@ export RACK_ENV="production"
 export SINATRA_ENV="production"
 export SEARCH_SERVER="https://elasticsearch.service.consul"
 export SEARCH_SERVER_ES7="https://elasticsearch.service.consul"
+export ELASTICSEARCH_PORT=443
+export ELASTICSEARCH_VERIFY_SSL=false
```
[the remark this answers is not in the document]
Comment (blarghmatey) [reply src/bilder/images/edxapp/templates/edxapp/mitx-staging/forum.env:16]: The reason it fails is due to us using the Consul DNS which doesn't match the CN specified in the auto-generated certificate. We can remove that if we switch the the AWS DNS name but wanted to just get things moving for now.

**Comment by blarghmatey** [comment]
It should definitely be possible to lock it down a bit based on the IAM role but didn't want to spend a bunch of time on that at the moment.

**Comment by shaidar** [comment]
Not excited about the permissive config but 👍

**Comment by blarghmatey** [comment]
Agreed, but it's not any different than the current setup with our self-managed ES clusters.

**Pull request merged**